package.json

```diff
@@ -140,7 +140,7 @@
     "test-unit:coverage": "jest --coverage",
     "test-unit:coverage-ci": "jest --coverage --maxWorkers 1 && codecov",
     "test-unit:watch": "jest --watch",
-    "test-e2e": "cypress run --browser chrome",
-    "test-e2e:watch": "cypress open"
+    "test-e2e": "cypress run --browser chrome -e EXAMPLES=$INSTALL_GUTENBERG_EXAMPLES",
```

Comment:
to avoid having to pass this explicitly, I've used `cypress_*` convention for other environment vars (like username, password), should we use the same convention? maybe `cypress_with_examples`?

Reply:
I would be fine switching `EXAMPLES` to a `cypress_*` var, but my thinking with `$INSTALL_GUTENBERG_EXAMPLES` was that it could be consistent for both installing and running the examples.